app/assets/javascripts/utils/date_utils.js

```diff
@@ -8,5 +8,48 @@ export const toDate = (date) => {
   if (date instanceof Date) {
     return date;
   }
-  return parseISO(date);
+  // should not really happen in production, but it's here for making the tests pass
```

Comment:
Hmm, I think this affects one or two React tests so shouldn't be too difficult to fix. Should I just raise an Error if this ever happens? That would fail the CI so if it is happening right now we would have an idea where and how.

Reply:
Yes, I think raising a console error (and then returning the new Date, so that it would still work the same way as with moment.js) would be a good way to flush out any places where this happens in production.